governance/reviews/template.md

```diff
@@ -23,171 +11,207 @@ Exemplary: project has an extraordinary level of governance development and impl
 Satisfactory: project has appropriate governance for its maturity level and is following that governance
 Mostly Satisfactory: project has mostly appropriate governance, but needs to fix one or two things
 Needs Work: project's governance is lacking and inadequate for its current level of maturity, and needs substantial work to overcome that
-
-NOTE: Fill this part as a summary of your review. It is recommended to start from the "Review" section below in the template.
+In Crisis: project appears to be having a meltdown and needs CNCF intervention to survive
 --->
 
-Status: Exemplary|Satisfactory|Mostly Satisfactory|Needs Work
+Status: Exemplary|Satisfactory|Mostly Satisfactory|Needs Work|In Crisis
 
-<!--- Short paragraph summarizing the general state of project governance. It should provide a final assessment status of one of the following: Satisfactory, Needs improvement, or Requires Attention. In the event the project governance requires attention, notify the TOC liaison for their awareness. --->
+<!--- Short paragraph summarizing the general state of project governance. In the event the project governance requires attention, notify the TOC liaison for their awareness. --->
 
 ### Executing the Assessment
 
-<!--- A brief description that details the timebox the assessment occurred and the individuals involved in the assessment. 
+<!--- A brief description that details the timebox the assessment occurred and the individuals involved in the assessment. --->
 
-Make sure you use a snapshot of the governance documents for your assessment and note the commit hash of the snapshot here as a link.
---->
-
-### Critical Items
+### Must-Fix Items
 
 The following issues have been identified that need to be resolved before [project milestone or other requirement]:
-<!--- The items in the list should be summarized, have a prioritized ordering and are expected to be considered blockers to project advancement. For each item in this list, a corresponding detailed description should be placed in the Findings table. Note that which items are required depends on the project's maturity level.
 
-Items in the "Governance Findings Table" at the bottom with "Critical" importance should be reported here.
+<!--- The items in the list should be summarized, have a prioritized ordering and are expected to be considered Critical or High, they are usually blockers to project advancement. For each item in this list, a corresponding detailed description should be supplied. Note that which items are required depends on the project's maturity level. 
```

Comment:
```suggestion
<!--- The items in the list should be summarized, have a prioritized ordering and are expected to be considered Critical or High, these are explicit and implicit blockers to project advancement within the [CNCF Matriculation process](https://github.com/cncf/toc/tree/main/process#how-to-apply-to-move-levels). For each item in this list, a corresponding detailed description should be supplied. Note that which items are required depends on the project's maturity level. 
```

Reply:
suggesting explicit and implicit be added here. Explicit being named criteria, implicit being something happening that inhibits or fails to meet the intent of the explicit criteria which would otherwise appear met.